test/server/cardImplementations/01_SOR/Shield.spec.js

```diff
@@ -0,0 +1,88 @@
+describe('Shield', function() {
```

Comment:
Can we add a TODO for interactions with Saboteur? Or would that go in the eventual Saboteur test?

Reply:
Yeah since that's specifically part of Saboteur I was planning to put it there